src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/Create/addresses.html.twig

```diff
@@ -37,14 +37,14 @@
         </div>
       </div>
     </div>
-    <div class="row" id="addresses-content">
+    <div class="row d-none" id="addresses-content">
       <div class="col">
         <span>{{ 'Delivery'|trans({}, 'Admin.Global') }}</span>
         <select class="form-control js-address-select" id="delivery-address-select"></select>
 
         <div class="card mt-3">
           <div class="card-body clearfix">
-            <button type="button" class="btn btn-primary float-right">{{ 'Edit'|trans({}, 'Admin.Actions') }}</button>
+            <a href="" class="btn btn-outline-primary float-right" id="js-delivery-address-edit-btn">{{ 'Edit'|trans({}, 'Admin.Actions') }}</a>
```

Comment:
Isn't it better to use button for those javascript actions? It prevents returning false in the click handler

Reply:
If you don't have link, don't use `a` tag, as @jolelievre says, use button. And it's not valid to have an empty link.

Reply:
hmm I add href during rendering in javascript side, its `a tag` so i don't have to add additional click listeners and stuff.. Is it still better to do a button?

Reply:
Ok, in this case `a` is the good choice but you can remove the empty `href` here since you fill it later